src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![doc(html_root_url = "https://docs.rs/slotmap/0.3.0")]
 #![crate_name = "slotmap"]
 #![cfg_attr(feature = "unstable", feature(untagged_unions))]
+#![cfg_attr(not(feature = "std"), feature(alloc))]
```

Comment:
BTW: I had `no_std` last night, but that wasn't cargo idiomatic as cargo features are additive, so `std` is more correct (and common).